api/graphql/schema.graphql

```diff
@@ -1795,7 +1794,7 @@ type Mutation {
     updateUserRolesInput: UpdateUserRolesInput! @spread
   ): UserAuthInfo
     @update(model: "User")
-    @canModel(ability: "updateRoles", model: "User")
```

Comment:
We should also add a @guard directive

Reply:
Sure https://github.com/GCTC-NTGC/gc-digital-talent/pull/10791/commits/b909155c77daf2abef96e43b67597dae1811d21f